### PR TITLE
cleanup spec for autorest

### PR DESCRIPTION
**Pull request by rikogeln-havok**
Cleaning up spec 
- autorest markup
- Adds api-version query parameter
- Removes malformed swagger examples

### Changelog
Please ensure to add changelog with this PR by answering the following questions.
  1. What's the purpose of the update?    
      - [X] new service onboarding 
      - [ ] new API version 
      - [ ] update existing version for new feature 
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify 
  2. When you are targeting to deploy new service/feature to public regions? Please provide date, or month to public if date is not available yet.
  3. When you expect to publish swagger? Please provide date, or month to public if date is not available yet.
  4. If it's an update to existing version,  please select SDKs of specific language and CLIs that require refresh after swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No, no need to refresh for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json [hunk 1]
```diff
@@ -1178,46 +953,62 @@
...
           "type": "string",
-          "format": "uri"
+          "description": "The URI of the Azure blob storage container containing the input model.",
+          "format": "uri",
+          "example": "https://contosostorage01.blob.core.windows.net/arrInput"
         },
         "storageContainerReadListSas": {
-          "type": "string"
+          "type": "string",
+          "description": "A Azure blob storage container shared access signature giving read and list access to the storage container. Optional. If not is not provided the Azure Remote Rendering rendering account needs to be linked with the storage account containing the blob container."
```
Comment (MalcolmTyrrell) [inline specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json:962]: Typo: An Azure

**Review by anuchandy — APPROVED** [review]

File: specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json (hunk 1 above)
Comment (rikogeln) [reply specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json]: fixed

File: specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json [hunk 2]
```diff
@@ -1284,49 +1082,67 @@
...
+        "message"
+      ]
     },
     "session_properties": {
       "properties": {
         "id": {
           "example": "contoso-session-c2b8ac1d-9494-4852-a59c-8c28813adc28",
-          "type": "string"
+          "type": "string",
+          "description": "The id of the session supplied when the conversion was created."
```
Comment (MalcolmTyrrell) [inline specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json:1111]: Conversion should be session.

File: specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json [hunk 3]
```diff
@@ -1284,49 +1082,67 @@
...
         },
         "target": {
-          "description": "The target of the error",
-          "type": "string"
+          "description": "The target of the particular error (e.g., the name of the property in error).",
+          "type": "string",
+          "readOnly": true
         },
-        "innererror": {
+        "innerError": {
```
Comment (MalcolmTyrrell) [inline specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json:1093]: This needs to be x-nullable, as does "details".

Comment (rikogeln) [reply specification/mixedreality/data-plane/Microsoft.MixedReality/preview/2021-01-01-preview/mr-arr.json:1093]: we will fix this on the service side by not emitting null values there

**Comment by anuchandy** [comment]
The API is in preview; and SDK dev is in progress; no SDK based on this api-version is released yet. The flagged breaking changes due to improvements and updates happened in the service side.

**Pull request merged**